src/test/java/edu/kis/powp/jobs2d/events/SelectComplexKite.java

```diff
@@ -0,0 +1,28 @@
+package edu.kis.powp.jobs2d.events;
+
+import edu.kis.powp.command.CommandFactory;
+import edu.kis.powp.command.ComplexCommand;
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+import edu.kis.powp.jobs2d.features.DriverFeature;
+import edu.kis.powp.jobs2d.magicpresets.FiguresJoe;
+import edu.kis.powp.command.*;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class SelectComplexKite implements ActionListener {
+    private DriverManager driverManager;
+    private ComplexCommand complexKite = new ComplexCommand();
+    private CommandFactory kiteFactory = new CommandFactory();
+
+
+    public SelectComplexKite(DriverManager driverManager) {
+        this.driverManager = driverManager;
+    }
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        ComplexCommand drawComplexKite = new ComplexCommand();
+        drawComplexKite.addCommand(new CommandFactory().makeKite(driverManager.getCurrentDriver()));
```

Comment:
nesting Kite into drawComplexKite is not necessary, but it should not be a problem also.

Now you should execute the command!